### ISSUE TITLE
[Feature Request] Volume Normalizer

### ISSUE THREAD

**Issue by lihengcao** [adrinu/adriMUSIC#2]
Hey! Some songs have different volumes, and it can be tedious to adjust the volume for each song or, worse, painful when a quiet song is followed by a loud song. 
It would be nice to have an audio-normalizing feature!

**Comment by adrinu** [comment]
I don't believe it is possible. The audio is directly streamed. The only way to do to download the file (or cache) and do some pre-processing to determine the correct audio output.
